examples/solids/src/cloptions.c

```diff
@@ -257,7 +257,7 @@ PetscErrorCode ProcessPhysics(MPI_Comm comm, Physics phys, Units units) {
   // Define derived units
   units->Pascal = units->kilogram / (units->meter * PetscSqr(units->second));
 
-  // Scale E to GPa
+  // Scale E to Pa
```

Comment:
Adjusted comment, though code runs faster if we scale to GPa (unless we were supposed to be using GPa all along?)

Reply:
I had [added](https://github.com/ArashMehraban/CeedPetscSolid/commit/d2c34594bc73c6af6edd7d155b09d1c5a5782abb) that comment when I introduced the scaling. I thought we were indeed using GPa because it's a typical scale for the materials considered. But then that was [scaled back](https://github.com/ArashMehraban/CeedPetscSolid/commit/765810ee4a0d9ef311be90df74196d752bec2cde) and the comment not updated.

Reply:
`-units_kilogram 1e-3 -units_second 1e3` would cause us to effectively use E in GPa internally. 1000 kg is a natural unit for mass of one cubic meter of material, and a characteristic wave speed of 1 km/s suggests a relevant time unit of ms (with length scale of 1 m). Our problems are time-independent so there are other ways to reach the same scaling.